polymetis/tests/scripts/7_joint_velocities.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import time
+
+import torch
+
+from polymetis import RobotInterface
+from utils import check_episode_log
+
+
+def test_new_joint_vel(state_log, joint_vel_desired):
+    joint_vel = state_log[-1].joint_velocities
```

Comment:
Since the velocity controller aims to maintain the desired velocity for the duration of the controller, you should check all velocities within the state log instead of just the last one. The first few steps can be left out as it's normal for the controller to need time to stabilize.